dstep/translator/MacroDefinition.d

```diff
@@ -2053,6 +2053,15 @@ bool translateFunctAlias(
     {
         Identifier ident = cast(Identifier) expr.expr;
 
+        if (ident.spelling == "assert")
+        {
+            // aliasing `assert` from C "assert.h" is commong enough to warrant
+            // a special case to use function instead of an alias
+            output.singleLine("void %s(T...)(T args) { assert(args); }",
```

Comment:
* It's only possible to pass two arguments two `assert`, a boolean condition and a string
* Please don't use this compact syntax for the generated function